addons/policies/bootstrap_entity_policies.json

```diff
@@ -3701,6 +3701,78 @@
                     "entity-delete"
                 ]
             }
+        },
+        {
+            "typeName": "AuthPolicy",
+            "attributes":
+            {
+                "name": "DATA_DOMAIN_AND_PRODUCT_BUSINESS_UPDATE_METADATA",
+                "qualifiedName": "DATA_DOMAIN_AND_PRODUCT_BUSINESS_UPDATE_METADATA",
+                "policyCategory": "bootstrap",
+                "policySubCategory": "default",
+                "policyServiceName": "atlas",
+                "policyType": "allow",
+                "policyPriority": 1,
```

Comment:
Do we need priority as 1?
1 is the higher priority

Impact of priority as 1 policy here would be that, no one can DENY any admin certain Domain Access via Persona Domain DENY policy